src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -184,7 +187,9 @@ class ElytraFlight : Module() {
     /* End of Control Mode */
 
     override fun onUpdate() {
-        if (mc.player == null || mc.player.isSpectator) return
+        /*Elytra check*/
+        elytraIsEquipped = mc.player.inventory.armorInventory[2].getItem() == Items.ELYTRA
+        if (!elytraIsEquipped || mc.player == null || mc.player.isSpectator) return
```

Comment:
You check if the player is null on line 192 but use the player in a non null context on line 191, this doesnt seem like such a good idea